src/modules/navigator/rtl_params.c

```diff
@@ -124,15 +124,18 @@ PARAM_DEFINE_INT32(RTL_TYPE, 0);
  * Defines the half-angle of a cone centered around the destination position that
  * affects the altitude at which the vehicle returns.
  *
+ * This is only supported for multicopters, since it has the agility to track altitudes
+ * consistently. For fixed wings,
+ *
  * @unit deg
  * @min 0
  * @max 90
- * @value 0 No cone, always climb to RTL_RETURN_ALT above destination.
+ * @value 0 No cone, always climb to RTL_RETURN_ALT above destination. -> This isn't true?
```

Comment:
This isn't true. There's another logic where another arbitrary minimum altitude boundary is applied (of 2 * acceptance_radius). But I guess that condition never really happens?

https://github.com/PX4/PX4-Autopilot/pull/19971/files#diff-c9eb22eb3f18ec6f5893df161720bdeecc56c8bf0164cbb7622a02edceb153e9R682-R686

Reply:
This condition happens when very close to the landing (within acceptance radius) already on triggering RTL no? So how about:
`No cone, climb to RTL_RETURN_ALT if horizontal travel is required.`
If you want to raise awareness for the behavior in close proximity of the destination then I would do that in the general param description above, as it applies to all options if I'm not mistaken.